packages/mcp-utils/src/server.ts

```diff
@@ -175,7 +175,26 @@ export type InitData = {
   clientCapabilities: ClientCapabilities;
 };
 
+type ToolCallBaseDetails = {
+  name: string;
+  arguments: Record<string, unknown>;
```

Comment:
I know `arguments` is probably the technically correct term, but wondering if we should stick to `parameters` for consistency? The official SDK calls these [`params`](https://github.com/modelcontextprotocol/typescript-sdk/blob/1d475bb3f75674a46d81dba881ea743a763cbc12/src/client/index.ts#L430) in the context of the client calling a tool.

Not overly opinionated though if you still prefer `arguments`.